frontend/public/components/utils/dropdown.jsx

```diff
@@ -289,11 +289,11 @@ export class Dropdown extends DropdownMixin {
     const {active, autocompleteText, selectedKey, items, title, bookmarks, keyboardHoverKey, favoriteKey} = this.state;
     const {autocompleteFilter, autocompletePlaceholder, noButton, noSelection, className, menuClassName, storageKey, canFavorite, dropDownClassName} = this.props;
 
-    const button = noButton
-      ? <div onClick={this.toggle} className="dropdown__not-btn" id={this.props.id}>
+    const button = (noButton) ?
```

Comment:
Our typical pattern is to put the ? and the : on the same line as the output.  Making this change will reduce the diff.

Reply:
And omitting the parentheses around `noButton` will further reduce the diff.